processor/priorityqueue.go

```diff
@@ -66,3 +69,72 @@ func (pq *pqT) Print() {
 		pkgLogger.Debug(*v)
 	}
 }
+
+type TransformRequestT struct {
```

Comment:
Can we remove the dependency with this struct in this file ? Not needed, but if there is a better way, we should do it.